automation/README.md

```diff
@@ -27,16 +26,28 @@ It may be used both locally and through CI.
 
   It also handles the cleanup of the container and nets (stop,rm).
 
-- run-integration-tests.mounts: Includes mounts to be used by the
-  oVirt CI (STDCI) worker.
+- run-tests.mounts: Includes mounts to be used by the oVirt CI (STDCI) worker.
 
-- run-integration-tests.packages: Includes the packages needed by
-  the oVirt CI (STDCI) worker.
+- run-tests.packages: Includes the packages needed by the oVirt CI (STDCI)
+  worker.
 
 ## Running the Tests
-Assuming *docker* is installed on the host,
-just run:
-`./automation/run-integration-tests.sh`
+Assuming *docker* is installed on the host, just run:
+`./automation/run-tests.sh`
+
+By default, `./automation/run-tests.sh` will run integration test only in
+container. You may change the test type by using:
+
+ * `TEST_TYPE=unit_code_style ./automation/run-tests.sh`: Code style tests.
```

Comment:
It will be nice to use a short name.
You can just use the term `lint` or `linter`.

And it is not limited to code stile, it finds bugs and missuses. I would name it `static analysis of code`.

Reply:
Done